2022/covid-19.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: JBCNConf - COVID-19 Policy
+bodyClass: home page-template-default page page-id-7 eventstation-class  eventstation-shop-column-4 wpb-js-composer js-comp-ver-5.0.1 vc_responsive
+---
+{% include base.html %}
+<div class="content-heading">
+  <div class="container-fluid">
+    <div class="heading-navigation">
+      <i class="fa fa-address-book"></i>
+      <ul>
+        <li>Home</li>
+        <li>&#47;</li>
+        <li>COVID-19 Policy</li>
+      </ul>
+    </div>
+
+    <div class="heading-title">
+    </div>
+  </div>
+</div>
+<div class="site-sub-content clearfix">
+  <div class="container-fluid">
+    <div class="row alternative-row">
+      <div class="col-lg-9 col-sm-9 col-xs-12 site-content-left">
+        <div class="category-post-list post-list single-list">
+          <article
+            class="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment"
+            id="post-294">
+            <div class="post-wrapper">
+              <div class="post-header">
+                <ul class="post-information">
+
+                </ul>
+                <h2 style="margin-top:50px;">COVID-19 Policy</h2>
+                For all our events, including conferences, workshops, and social events, we take the COVID-situation
+                seriously. We want to reduce the risk of infection as much as possible, and keep everyone safe.
+
+                We will adhere to <a
+                href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/#bloc7">local government
+                regulations</a> and guidelines, and when deemed necessary, take
+                additional measures at our discretion. As regulations and best practices around COVID management for
+                events evolve constantly, we may change our approach up to the last moment.
+
+                The latest version of this policy is available at <a href="https://www.jbcnconf.com/2022/covid-19.html">COVID-19
+                Policy</a>.
+
+                If you have any questions or concerns about our COVID-19 policy, please do not hesitate to contact us
+                via <a href="mailto:info@barcelonajug.org">info@barcelonajug.org</a>.
+
+                <h3>What we expect from you</h3>
```

Comment:
I would include a section similar to this one or in the top of the page (up to you) where we can include a message saying when it was the last time this page was changed and which measures we have to follow at that time. Something like:
```
News
-Update 25/4/22:
Since 20/4/22, it is no longer mandatory to wear masks in enclosed spaces in Spain.
```